database/src/migrations/release.0.20/biohub.sql

```diff
@@ -729,6 +730,8 @@ CREATE TABLE permit(
 
 COMMENT ON COLUMN permit.permit_id IS 'System generated surrogate primary key identifier.'
 ;
+COMMENT ON COLUMN permit.system_user_id IS 'System generated surrogate primary key identifier.'
+;
```

Comment:
We'll need to make sure Charlie knows to incorporate this in his tool, otherwise the next time he does a release, this might get overwritten.

Reply:
Alternatively, could add this as a regular migration, as it will be more obvious that its a change that needs incorporating.

Reply:
ive informed him, should be all good